samples/interactive-tutorials/src/main/java/events/setup/RemoveEventsResources.java

```diff
@@ -28,19 +28,17 @@
 import com.google.cloud.retail.v2.ProductServiceClient.ListProductsPagedResponse;
 import java.io.IOException;
 
-public class RemoveTestResources {
+public class RemoveEventsResources {
 
-  public static void main(String... args) throws IOException {
-    // TODO(developer): Replace these variables before running the sample.
+  public static void main(String[] args) throws IOException {
     String projectId = ServiceOptions.getDefaultProjectId();
-    String bucketName = System.getenv("BUCKET_NAME");
+    String bucketName = System.getenv("EVENTS_BUCKET_NAME");
     String branchName =
         String.format(
             "projects/%s/locations/global/catalogs/default_catalog/branches/0", projectId);
 
     deleteBucket(bucketName);
     deleteAllProducts(branchName);
```

Comment:
Should we remove this? It should be in RemoveProductsResources.java

Reply:
Added deleteAllEvents.